### ISSUE TITLE
Feature Request: TFLite RESIZE_BILINEAR quantization

### ISSUE THREAD

**Issue by ajarthurs** [tensorflow/tensorflow#28269]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
Arch Linux 5.0.7

- TensorFlow installed from (source or binary):
source

- TensorFlow version (or github SHA if from source):
5d3aa3663a7a07e9f6494ec86fee5be44aa00597

**Provide the text output from tflite_convert**
Using `tensorflow/lite/tools/optimize/calibration` via Python for calibration-and-quantization. Reports the following after calibration, during quantization.
```
RuntimeError: Quantization not yet supported for op: RESIZE_BILINEAR
```

Also, please include a link to a GraphDef or the model if possible.
`tensorflow/tpu/models/official/retinanet` is one example model containing RESIZE_BILINEAR.

**Any other info / logs**

Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by ajarthurs** [comment]
I am attempting to quantize the RetinaNet model using TFLite's calibration optimization tool. A copy of a floating-point (32bit) RetinaNet TFLite model, trained on MSCOCO, can be found at https://www.dropbox.com/s/hnkmqzcasb8lu8n/retinanet-float32.tflite?dl=0

[calibrate_and_quantize.py](https://github.com/tensorflow/tensorflow/files/3131714/calibrate_and_quantize.py.zip) is a Python script that runs TFLite's calibration tool to quantize the said model. To run it:
```
# Adjust paths as needed.
$ PYTHONPATH=path/to/tensorflow/tpu/models/official/retinanet:$PYTHONPATH python calibrate_and_quantize.py --input_tflite_file=retinanet-float32.tflite --output_tflite_file=retinanet-int8.tflite --train_file_pattern=path/to/mscoco-tfrecords/train*
```

The script will end with an error stating: `RuntimeError: Quantization not yet supported for op: RESIZE_BILINEAR`.

A floating-point TFLite implementation of `RESIZE_BILINEAR` is in `tensorflow/lite/kernels/resize_bilinear.cc`.

**Comment by ajarthurs** [comment]
Included in 79fca9af73b2634a64d10a0caad4e35bc6ca00d1

**Comment by ajarthurs** [comment]
Closing since `79fca9a` enables `RESIZE_BILINEAR` to quantize, u/int-8 support added since `fc4005be8eed17eccce613b52888c0b783565800`
